AirCasting/Graph/GraphView.swift

```diff
@@ -10,30 +10,33 @@ import SwiftUI
 struct GraphView: View {
     
     let session: SessionEntity
-    let thresholds: [SensorThreshold]
+    let threshold: SensorThreshold
+    @Binding var selectedStream: MeasurementStreamEntity?
```

Comment:
binding UI against `NSManagedObject` subclasses is suboptimal, but I think we'll be addressing this holistically across the app in near future so we can leave it for now